README.md

```diff
@@ -5,7 +5,7 @@ The official repo for the JumboCode website (and project).
 Project Lead: Olivia MacDougal
 
 Team Members: 
-* Alice Dempsey
+* Alice Dempsey - hey y'all
```

Comment:
Since this was just for practice, can you delete this addition? We want to keep master clean, even if it is just the Readme.